README.md

```diff
@@ -1 +1,58 @@
-# microverse-m1-w1-d3-portfolio-production
\ No newline at end of file
+# Microverse Portfolio Progect
+## Milestone 1
+
+Build a developer portfolio.
+
+Template includes all sections required for presenting your projects and providing information to employer.
+
+
+
+## Built With
+
+- CSS HTML
+- Git version control system, Github workflows
+
+
+
+## Getting Started
+
+Clone the repository
+
+```
+git clone https://github.com/repository
+```
+Create your feature branch
+
+```
+git checkout -b <new-branch>
+```
+
+You can use default Microverse linters set or configure linting rules
+
+Expand .gitignore with more exceptions if needed
+(add test files with .extension you'd like to be ignored by Git)
+
+Customize template portfolio design
+Push your repository to GitHub
+
+```
+git add .
+git commit -m "Commit description"
+git push origin <your-branch>
+```
+Open pull request
+
+Check Linters:
+- Lighthouse
+- Webhint
+- Stylelint
+
+Debug HTML and CSS files if checks were unsuccessful
+
+
+
+## Author
+
+👤 **Nadia Levina**
+
+- GitHub: [@petushka1](https://github.com/petushka1)
```

Comment:
- [ ] README file is almost ready, I would recommend adding these missing sections:

- Contributing
- Show your support
- Acknowledgements
- License

Here's a [template](https://github.com/microverseinc/readme-template/blob/master/README.md) you can use.